distribution/src/main/resources/bin/elasticsearch-env.bat

```diff
@@ -28,6 +28,13 @@ if not exist %JAVA% (
   exit /B 1
 )
 
+rem don't let JAVA_TOOL_OPTIONS slip in (e.g., agents in Ubuntu so this works
+rem around https://bugs.launchpad.net/ubuntu/+source/jayatana/+bug/1441487)
+if not "%JAVA_TOOL_OPTIONS%" == "" (
```

Comment:
I'd be more comfortable if it was documented in the "Install Elasticsearch with zip on Windows". We have few notes there already.